third_party/blink/renderer/core/inspector/inspector_dom_debugger_agent.cc

```diff
@@ -969,6 +969,8 @@ void ReplayNotifyBeforeEvent(const String& eventName,
       MakeReplayEventType(eventName, eventTarget, isCallback);
 
   // Disabled by default, see https://linear.app/replay/issue/RUN-1251
+  recordreplay::Assert("[RUN-1226] ReplayNotifyBeforeEvent %d %s",
+                       gIsEventInFlight, replayEventType.Ascii().c_str());
```

Comment:
This should be under the FeatureEnabled test, so we don't get spurious mismatches from this when event collection is disabled.